.changeset/big-elephants-add.md

```diff
@@ -0,0 +1,5 @@
+---
+"@clerk/shared": patch
+---
+
+Expose match from `path-to-regexp`.
```

Comment:
Let's also highlight that this is an internal change without user-facing impact. Suggestion, feel free to tweak it:

```suggestion
Exports `match` utility from the `path-to-regexp` lib.
```